### DISCUSSION TITLE
你好，ADO Query<T> 不支持Column定义的Name映射吗？

### DISCUSSION THREAD

**Discussion by zenghui** [dotnetcore/FreeSql#1365] [General]
fsql.Ado.Query<User>("select * from account_user")

[Table(Name = "account_user")]
public class User
{
    [Column(IsPrimary = true, IsIdentity = true)]
    public long ID { get; set; }

    [Column(Name = "update_at")]
    public DateTime UpdateAt { get; set; }
}

**Comment by 2881099** [comment]
是的，ado 不属于映射范围，请使用 sql as 解决。

**Reply by zenghui** [reply]
谢谢， 有类似Dapper.DefaultTypeMap.MatchNamesWithUnderscores = true;这种吗

**Reply by 2881099** [reply]
暂时没有，Ado 不是主要功能

**Reply by 2881099** [reply]
侧重点不一样

**Reply by 2881099** [reply]
用 fsql.Select\<dto\>().WithSwl(sql).ToList() 应该可行，提前设置 FreeSqlBuilder().UseNameConvert(...看这里的枚举定义注释)
